### PR TITLE
feat(statics): add more sol token

### PR DESCRIPTION
**Pull request by ewangbitgo**
TICKET: EA-1955

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -10288,6 +10622,7 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.SLND,
     AccountCoin.DEFAULT_FEATURES
   ),
+
```
Comment (mishkinf-bitgo) [inline modules/statics/src/coins.ts:10625]: not a big issue but this space is not necessary

**Review by mishkinf-bitgo — DISMISSED** [review]
lgtm

File: modules/statics/src/base.ts [hunk 2]
```diff
@@ -1324,6 +1324,42 @@ export enum UnderlyingAsset {
...
+  'tuAtlas' = 'tuAtlas',
+  'tuSAMO' = 'tuSAMO',
+  'tuDFL' = 'tuDFL',
+  'tuLIKE' = 'tuLIKE',
+  'tuPOLIS' = 'tuPOLIS',
+  'tuSRM' = 'tuSRM',
+  'tuREAL' = 'tuREAL',
+  'tuMBS' = 'tuMBS',
+  'tuGENE' = 'tuGENE',
+  'tuCOPE' = 'tuCOPE',
```
Comment (CapnMigraine) [inline modules/statics/src/base.ts:1361]: Could we use lowercase for all token names? these combinations seem a bit arbitrary
Other sol tokens use uppercase for the key and lowercase for the value

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -10207,6 +10207,340 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.QCAD,
     AccountCoin.DEFAULT_FEATURES
   ),
+
+  solToken(
+    '170073d0-3641-42d1-b44a-73509e8bfb23',
+    'SOL:SHDW-USDC',
```
Comment (CapnMigraine) [inline modules/statics/src/coins.ts:10213]: I think these should be all lower case

**Review by alebusse — APPROVED** [review]

**Pull request merged**